packages/client/src/components/SendTokens/components/AddressInput.js

```diff
@@ -28,23 +19,12 @@ const AddressInput = () => {
       <label className="has-text-grey mb-2">
         Address <span className="has-text-red"> *</span>
       </label>
-
-      <div className="is-flex">
-        <div className="flex-1" style={{ position: "relative" }}>
-          <input
-            style={{ height: 48 }}
-            className="border-light rounded-sm column is-full p-2 mt-2"
-            type="text"
-            value={recipient}
-            onChange={onrecipientChange}
-          />
-          {recipientValid && (
-            <div style={{ position: "absolute", right: 17, top: 20 }}>
-              <Svg name="Check" />
-            </div>
-          )}
-        </div>
-      </div>
+      <InputAddress
```

Comment:
This input Address component is being replaced by addressDropdown. I think we can safely delete this file now https://github.com/DapperCollectives/VESSEL/pull/120/files#diff-e8e43eadee51d689965c6c2bd4a914f301aea66e12351d018baa6cd3567d1422